core/src/main/java/org/p2p/core/wrapper/eth/rlp/RLP.kt

```diff
@@ -303,15 +329,22 @@ object RLP {
                     }
 
                     val rlpData = ByteArray(lengthOfLength.toInt() + length + 1)
-                    System.arraycopy(msgData, pos, rlpData, 0, lengthOfLength.toInt()
-                            + length + 1)
+                    System.arraycopy(
+                        /* src = */ msgData,
+                        /* srcPos = */ pos,
+                        /* dest = */ rlpData,
+                        /* destPos = */ 0,
+                        /* length = */ lengthOfLength.toInt() + length + 1
+                    )
 
                     if (level + 1 < depth) {
                         val newLevelList = RLPList()
                         newLevelList.rlpData = rlpData
 
-                        fullTraverse(msgData, level + 1, pos + lengthOfLength.toInt() + 1,
-                                pos + lengthOfLength.toInt() + length + 1, newLevelList, depth)
+                        fullTraverse(
```

Comment:
if you are scared that ETH module will stop working - i can add `internal` directory for ETH and add an exclude rule for ktlint